CONTRIBUTING.rst

```diff
@@ -60,6 +60,33 @@ Once you have done this, you can launch the master branch of Jupyter notebook
 from any directory in your system with::
 
     jupyter notebook
+  
+Verification 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    
+While running the notebook, select one of your notebook files (the file will have the extension .ipynb). 
+In the top tab you will click on "Help" and then click on "About". In the pop window you will see information about the version of Jupyter that you are running. You will see "The version of the notebook server is:". 
+If you are working in development mode, you will see that your version of Jupyter notebook will include the word "dev". 
+
+.. image:: https://user-images.githubusercontent.com/327925/38238574-8a1ada9e-372a-11e8-8a9c-2a74af743390.png
+   :width: 40pt
+
+If it does not include the word "dev", you are currently not working in development mode and should follow the steps below to uninstall and reinstall Jupyter. 
+
+Troubleshooting the Installation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you do not see that your Jupyter Notebook is not running on dev mode, it's possible that you are
+running other instances of Jupyter Notebook. You can try the following steps:
+
+1. Uninstall all instances of the notebook package. These include any installations you made using
+   pip or conda
+2. Run ``python -m pip install -e .`` in the notebook repository to install the notebook from there
```

Comment:
Let's assume that folks are using python3 and change the command to ``python3 -m pip install -e .``